packages/manager/modules/pci/src/projects/project/storages/cold-archive/archives/onboarding/onboarding.component.js

```diff
@@ -0,0 +1,8 @@
+import controller from './onboarding.controller';
+import template from './onboarding.html';
+
+export default {
+  controller,
+  template,
+  bindings: {},
```

Comment:
If property is empty you can probably remove it

Reply:
`bindings: {}` will have multiple properties in the next commit.